src/ai/backend/manager/models/vfolder.py

```diff
@@ -512,6 +513,12 @@ def vfid(self) -> VFolderID:
         return VFolderID(self.quota_scope_id, self.id)
 
 
+# Although the length constraint of the `vfolders.name` column is 128,
+# we limit the length to 64 in the create/rename API
+# because we append a timestamp of deletion to the name when VFolders are deleted.
+VFolderNameTrafaret = tx.Slug(allow_dot=True, max_length=64)
```

Comment:
Please update all `64` constants representing the length of the vfolder name to use a defined constant.
Also, kindly replace all 128 magic number with a properly defined constant as well. @fregataa

Reply:
I removed this type definition since it is not used anywhere